src/krishna/Test1_NonRepeatingChar.java

```diff
@@ -0,0 +1,29 @@
+package krishna;
+
+public class Test1_NonRepeatingChar {
+
+	void fNonRepeat(String word) {
+		for(int i= 0; i<word.length(); i++) {
+			int temp = 0;
+			for(int j= 0; j<word.length(); j++) {
+				if(word.charAt(i) == word.charAt(j)) 
+					temp++;
+			}
+			if(temp ==1) {
```

Comment:
here you can aslo use 
indexOf(ch) ==lastIndexOf(ch) logic to avoid one more for loop